src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -2389,10 +2388,9 @@ namespace SurfaceGeometry {
                 for (int SurfNum = state.dataHeatBal->Zone(ZoneNum).HTSurfaceFirst; SurfNum <= state.dataHeatBal->Zone(ZoneNum).HTSurfaceLast;
                      ++SurfNum) {
                     if (state.dataSurface->Surface(SurfNum).Class == SurfaceClass::Floor) {
-                        state.dataHeatBal->Zone(ZoneNum).FloorArea += state.dataSurface->Surface(SurfNum).Area;
                         state.dataHeatBal->Zone(ZoneNum).HasFloor = true;
                         int spaceNum = state.dataSurface->Surface(SurfNum).spaceNum;
-                        state.dataHeatBal->space(spaceNum).floorArea += state.dataSurface->Surface(SurfNum).Area;
+                        state.dataHeatBal->space(spaceNum).calcFloorArea += state.dataSurface->Surface(SurfNum).Area;
```

Comment:
Off topic but I wonder what would happen if user entered calcFloorArea != user entered CeilingArea

Reply:
I pulled this branch and reviewed there since this view is hard to follow. I don't see anything that stands out as a problem. Why are the space.calcFloorAreas summed (above) and the ZONE.CeilingArea summed (below). What if the ceiling is split by space? I would have thought the floor and ceiling calculations would be similar wrt space and zone.

Reply:
I guess a ceiling split by space would just be 2 ceiling surfaces in the zone so they would get summed correctly. But then that would hold true for the floor as well. Since the issue was with respect to floor area this is probably a moot point (space floor area vs zone ceiling area).

Reply:
@rraustad Good questions. The main focus here was getting the internal gain levels correct, since they can be specified per floor area. At the moment, `state.dataHeatBal->space` only has fields related to floor area, nothing yet for ceiling height, ceiling area, or volume. That's future work.